scripts/exportrestapi.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+AWS_PROFILE="${AWS_PROFILE:-atat-sandbox-dev}"
```

Comment:
I don't know whether we necessarily want a default? A user that has not configured their `AWS_PROFILE` env var may also not use the same naming conventions we have documented privately in Confluence.

Reply:
So, you advise removing the default?